kani-driver/src/exe_trace.rs

```diff
@@ -0,0 +1,288 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use crate::session::KaniSession;
+use kani_metadata::HarnessMetadata;
+use std::collections::hash_map::DefaultHasher;
+use std::ffi::OsString;
+use std::fs;
+use std::hash::{Hash, Hasher};
+use std::path::Path;
+use std::process::Command;
+
+/// Module for parsing deterministic values from CBMC output traces,
+/// generating executable traces, and adding them to the user's source code.
+
+impl KaniSession {
+    /// The main driver for generating executable traces and adding them to source code.
+    pub fn exe_trace_main(&self, specialized_obj: &Path, harness: &HarnessMetadata) {
```

Comment:
Rename this to something like `generate_test` or per current schema `exe_trace`.

Reply:
In addition to generating an exe trace, this fn also adds it to the src code. So I renamed it to `gen_and_add_exe_trace`.